__tests__/app.test.js

```diff
@@ -167,4 +167,42 @@ describe("/api/articles/:article_id/comments", () => {
             expect(body.msg).toBe("bad request!")
         })
     })
+})
+describe("PATCH /api/articles/:article_id", () => {
+    test(":) PATCH 201 - updates the article given article id, returns updated article", () => {
```

Comment:
201 is usually reserved for POST requests where a new entry has been created. This is just a successful request.